docs/doc_versions.txt

```diff
@@ -0,0 +1,6 @@
+latest
+stable
```

Comment:
What does "stable" mean in the context of not yet having a Version 1.0?

Reply:
`stable` is supposed to be the latest *released* version (vs `latest` being bleeding edge from main branch). The idea is to have a permalink into the newest release, but right now, no commit automatically generates it. 

I am not sure if we can use the `tag` event to push at the same time to `stable` and assume that "last tag == stable release"

Reply:
Ah, so the intention is that 0.11.0 should be the same thing as "stable"?

Reply:
yep! I believe that's the way readthedocs does/names them as well.